website_js_below_the_fold/readme/USAGE.rst

```diff
@@ -2,4 +2,4 @@ This is a technical module and have no graphical interface whatsoever.
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
This should be deleted from README by fragments. It's already in the header of the generated README.

Reply:
done